spec/views/posts/post_index_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
```

Comment:
- [ ] Some of your tests are passing but there are two you need to fix 👇 

![image](https://user-images.githubusercontent.com/75126481/180192947-5bfddd38-ef4a-4dc6-bfed-2c96db7e33af.png)